app/assets/stylesheets/wizard.scss

```diff
@@ -0,0 +1,946 @@
+
+.wizard-miq-review-steps {
+  list-style: none;
+
+  .list-group, .list-group-item {
+    border: none;
+    margin-bottom: 0;
+  }
+
+  > ul {
+    > li {
+      position: relative;
+      float: left;
+      margin: 0;
+      line-height: 15px;
+      width: 100%;
+
+      > a {
+        color: #030303;
+        text-decoration: none;
+        font-size: 16px;
+        transition: 0.25s;
+        cursor: pointer;
+        padding-left: 30px;
+        padding-right: 5px;
+
+        &:before {
+          content: "\f107";
+          display: block;
+          font-family: FontAwesome;
+          font-size: 24px;
+          font-weight: 500;
+          position: absolute;
+          left: 20px;
+        }
+
+        &.collapsed {
+          &:before {
+            content: "\f105";
+          }
+        }
+      }
+
+      .checkbox {
+        line-height: 20px;
+        pointer-events: none;
+      }
+    }
+  }
+}
+
+.wizard-miq-review-substeps {
+  padding-left: 22px;
+  > ul {
+    > li {
+      position: relative;
+      float: left;
+      margin: 0;
+      line-height: 15px;
+      width: 100%;
+
+      a {
+        color: #030303;
+        text-decoration: none;
+        font-size: 16px;
+        transition: 0.25s;
+        cursor: pointer;
+        padding-left: 30px;
+        padding-right: 5px;
+
+        &:before {
+          content: "\f107";
+          display: block;
+          font-family: FontAwesome;
+          font-size: 24px;
+          font-weight: 500;
+          position: absolute;
+          left: 20px;
+        }
+
+        &.collapsed {
+          &:before {
+            content: "\f105";
+          }
+        }
+      }
+    }
+  }
+}
+
+.wizard-miq-review-content {
+  padding-top: 10px;
+  padding-left: 40px;
+
+  .miq-form-heading {
+    padding-top: 10px;
+
+    &:first-of-type {
+      padding-top: 0;
+    }
+  }
+  .wizard-miq-review-item {
+    padding: 5px 0;
+    &.sub-item {
+      margin-left: 10px;
+    }
+    .wizard-miq-review-item-label {
+      font-weight: 700;
+      padding-right: 10px;
+    }
+    .wizard-miq-review-item-value {
+    }
+    .wizard-miq-review-item-subvalue {
+      padding-left: 20px;
+    }
+    .wizard-miq-review-item-field {
+      font-weight: 700;
+      padding-right: 10px;
+      margin: 5px 0;
+      &:first-of-type {
+        margin-top: 0
+      }
+      &:last-of-type {
+        margin-bottom: 0;
+      }
+      &.sub-field {
+        margin-left: 10px;
+      }
+    }
+  }
+  .wizard-miq-review-subform {
+    padding-left: 15px;
+  }
+}
+
+.wizard-miq-footer {
+  padding-bottom: 17px;
+  background-color: #f8f8f8;
+}
+.wizard-miq-btn-container {
+  border-top: solid 1px #d1d1d1;
+  float: right;
+  width: 100%;
+  height: 60px;
+  padding: 15px;
+  margin-top: 0;
+
+  .tooltip-wrapper {
+    border: none;
+    box-shadow: none;
+    display: inline-block;
+    margin-left: 5px;
+    padding: 0;
+    text-align: center;
+
+    .btn[disabled] {
+      pointer-events: none;
```

Comment:
Note that this does not prevent the user from activating the button by keyword, and does break any `cursor` style and tooltips. Sure about it?